includes/functions/functions_email.php

```diff
@@ -281,46 +287,44 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
                     $mail->Port = 587;
                     $mail->Host = 'smtp.gmail.com';
                     $mail->Username = (!empty(trim($email_mailbox))) ? trim($email_mailbox) : EMAIL_FROM;
-                    if (trim($email_password) != '') {
+                    if (trim($email_password) !== '') {
```

Comment:
Should we start converting !== '' to !empty()?

Reply:
Nope, the $email_password is known to be a string and checking its trim for specifically an empty string is more performant.